src/main/java/org/folio/service/inventory/InventoryItemManager.java

```diff
@@ -250,6 +248,40 @@ public Future<List<Piece>> handleItemRecords(CompositePoLine compPOL, Location l
       });
   }
 
+  private Future<List<JsonObject>> searchStorageExistingItems(String poLineId, String holdingId, int expectedQuantity,
+                                                              RequestContext requestContext) {
+    String query = String.format(LOOKUP_ITEM_QUERY, poLineId, holdingId);
+    RequestEntry requestEntry = new RequestEntry(ITEM_STOR_ENDPOINT).withQuery(query).withOffset(0).withLimit(expectedQuantity);
+    return restClient.getAsJsonObject(requestEntry, requestContext)
+      .map(itemsCollection -> {
+        List<JsonObject> items = extractEntities(itemsCollection);
+        logger.debug("{} existing items found out of {} for PO Line with '{}' id", items.size(), expectedQuantity, poLineId);
```

Comment:
it would be great to start with method name like in other paces

Reply:
I saw other places, please use common style other places as well

Reply:
Please note that first commit in the PR is `move private methods with one usage below their callers`.
Although I agree with your suggestion we can't fix logging and other usages in all replaced code parts. From this perspective please review only consecutive commits

Reply:
Ok,